src/main/resources/logback.groovy

```diff
@@ -7,17 +7,76 @@ import ch.qos.logback.core.status.OnConsoleStatusListener
 import static com.ofg.config.BasicProfiles.PRODUCTION
 import static org.springframework.core.env.AbstractEnvironment.ACTIVE_PROFILES_PROPERTY_NAME
 
+class LogbackConfig {
+    private static String DEFAULT_FILENAME = 'logs/application.log'
+    private static String DEFAULT_LOG_PATTERN = '%d{yyyy-MM-dd HH:mm:ss.SSSZ, Europe/Warsaw} | %-5level | %X{correlationId} | %thread | %logger{1} | %m%n'
+    private static String DEFAULT_SCAN_TIME = '1 minutes'
+    private static String DEFAULT_ROLLING_FILENAME_PATTERN = 'logs/application.%d{yyyy-MM-dd}.log.zip'
+    private static int DEFAULT_ROLLING_MAX_HISTORY = 7
+    
+    private Properties globalProperties
+    
+    String getLogPattern() {
+        return globalProperty('logger.log.pattern') ?: DEFAULT_LOG_PATTERN
+    }
+
+    String getScanTime() {
+        return globalProperty('logger.scan.time') ?: DEFAULT_SCAN_TIME
+    }
+
+    String getRollingFilenamePattern() {
+        return globalProperty('logger.rolling.file.pattern') ?: DEFAULT_ROLLING_FILENAME_PATTERN
+    }
+
+    String getLoggerFilename() {
+        return globalProperty('logger.file') ?: DEFAULT_FILENAME
+    }
+
+    int getRollingMaxHistory() {
+        String maxHistory = globalProperty('logger.rolling.history.max')
+        if (maxHistory != null) {
+            return Integer.parseInt(maxHistory)
+        } else {
+            return DEFAULT_ROLLING_MAX_HISTORY
+        }
+    }
+
+    private File globalPropertiesFile() {
+        def env = System.getenv()
+        String configFolder = env['CONFIG_FOLDER']
+        return new File("${configFolder}/common/global.properties")

```

Comment:
I don't remember. Maybe it wasn't me or maybe I have too many threads to handle I just threw away your explanations from my mind ;). Anyway great that you checked it.